src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -116,7 +116,7 @@ trait NodeStatusBuilder {
 trait BlockChainBuilder {
   self: StorageBuilder =>
 
-  lazy val blockchain: Blockchain = BlockchainImpl(storagesInstance.storages)
+  lazy val blockchain: BlockchainImpl = BlockchainImpl(storagesInstance.storages)
```

Comment:
With our current codebase it is, as the `LedgerImpl` requires a `BlockchainImpl` as it is currently coupled with the `InMemoryWorldStateProxy` (which probably should be changed, I added a [FIXME](https://github.com/input-output-hk/etc-client/pull/284/files#diff-1b4e47475b368c25eb5141199a4289d0R25) regarding that).